inc/templates/namespace.php

```diff
@@ -530,12 +530,36 @@ function setup_head(
 /**
  * Manage any active integrations by inserting an `irving/integrations` component.
  *
- * @param array $data Data object to be hydrated by template.
- * @return array The update endpoint data.
+ * @param array     $data    Data object to be hydrated by templates.
+ * @param \WP_Query $query   The current WP_Query object.
```

Comment:
We have a use statement for WP_Query, so the slash is unnecessary.

```suggestion
 * @param WP_Query $query   The current WP_Query object.
```